lib/auth/config.ts

```
@@ -0,0 +1,143 @@
+     */
+    providers?: any[];
+  };
+}
+
+/**
+ * Default authentication configuration
+ */
+export const defaultAuthConfig: AuthConfig = {
+  provider: "next-auth",
+  supabase: {
+    url: process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)